src/common/icons/Remove.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
Is this different from the existing Trash.js? Do we need a separate one?

Reply:
It is slightly different. Trash.js is a trashcan and this is a circle with an X in the middle which matches the shape of the circle with a checkmark in the middle when a video is selected. Either could be used though.

Reply:
`TimesCircle` or `TimesCircleSolid` might be what you're after.

Reply:
Thank you both. I have removed this new icon and set the modal to use `TimesCircleSolid`.